data/wayland-proxy-virtwl.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Wayland-proxy-virtwl
+After=cosmic-session.target
+WantedBy=cosmic-session.target
+
+[Service]
+ExecStart=/var/bin/wayland-proxy-virtwl --wayland-display wayland-0 --x-display=0 --xrdb Xft.dpi:150 --x-unscale=2
```

Comment:
Do we want to hard code dpi and unscaling? I don't think so.

This is better handled by the user doing an override with `systemctl --user edit wayland-proxy-virtwl`.

Reply:
Also:
- Lets add `ExecStartPost=/usr/bin/systemctl --user set-environment DISPLAY=":0"`
- Also don't use `wayland-0`, that socket has a special fallback meaning. Rather use something custom, like `wayland-virtwl`
- Additionally we can also add and install section:
```
[Install]
WantedBy = cosmic-session.target
```

Reply:
ok thanks, i'll make the changes. I saw the dpi & unscaling as suggestions from the README, but ya users can probably override it themselves.

Reply:
And I was setting $DISPLAY in start-cosmic, but i'll do that here as you've recommended instead.